### ISSUE TITLE
CTCloss suddenly appears nan

### ISSUE THREAD

**Issue by BADBADBADBOY** [pytorch/pytorch#80794]
### 🐛 Describe the bug

Training text recognition code crnn, a total of 200 epoch training, at 150 epoch, loss suddenly appears nan

### my CTCloss code：

```
from torch.nn import CTCLoss as PytorchCTCLoss
import torch.nn as nn
class CTCLoss(nn.Module):
    def __init__(self, ):
        super(CTCLoss, self).__init__()
        self.criterion = PytorchCTCLoss(reduction='none')
    def forward(self, preds, labels, preds_size, labels_len):
        ## preds, text, preds_size, length
        preds = preds.log_softmax(2).requires_grad_()  # torch.ctcloss
        loss = self.criterion(preds, labels, preds_size, labels_len)
        print(loss)
        loss = loss.sum()
        return loss
```

### Versions

torch    1.6.0+cu101
torchvision    0.7.0+cu101

**Comment by ezyang** [comment]
Check out the docs for anomaly mode, and the forums  https://discuss.pytorch.org/ can offer more help for debugging nans. Without a reproducer or more evidence that an error in PT is inducing the NaN we won't be able to investigate.
